vbfml/scripts/err_analysis

```diff
@@ -783,5 +826,66 @@ def plot_correl(ctx, input_dir: str) -> None:
         f.write(table_print)
 
 
+@cli.command()
+@click.pass_context
+@click.option(
+    "-i",
+    "--input-dir",
+    required=True,
+    help="path to the prediction directory in a model",
+)
+@click.option(
+    "-c", "--cut-variable", required=True, help="The quantity to cut on the threshold."
+)
+@click.option(
+    "-t", "--threshold", type=float, required=True, help="The threshold/cut value."
+)
+@click.option(
+    "-p",
+    "--plot-variable",
+    default="score",
+    required=True,
+    help="The variable to plot the distribution after the cut",
+)
+def dist_cut(
+    ctx, input_dir: str, cut_variable: str, threshold: float, plot_variable: str
+) -> None:
```

Comment:
Can we rename this function? Maybe we can use something like `cut_and_plot`, if you think that's descriptive of the function

Reply:
By the way, for the `--plot-variable` argument above, is it strictly required? I see you're setting a default of `"score"` above (which is nice I think), so would this function work if `-p` option is not specified? If so, can you set `required=False` for that option just for consistency? (I'm not sure if that will cause issues in edge cases)